runtime/worker_bootstrap.rs

```diff
@@ -23,6 +27,31 @@ pub struct BootstrapOptions {
   pub user_agent: String,
 }
 
+impl Default for BootstrapOptions {
+  fn default() -> Self {
+    let cpu_count = thread::available_parallelism()
+        .map(|p| p.get())
+        .unwrap_or(1);
+
+    let runtime_version = env!("CARGO_PKG_VERSION").into();
+    let user_agent = format!("Deno/{}", runtime_version);
```

Comment:
Might want to change this to `Deno-runtime` or something, but I think most people would likely be fine with just a generic `Deno` user agent if they don't care much about the defaults.